CHANGES/2332.feature

```diff
@@ -0,0 +1 @@
+Implements a global changelog limit, to reduce the amount of duplicated changelog information that Pulp needs to keep track of. Controlled by a setting named `KEEP_CHANGELOG_LIMIT`, which defaults to 10.
```

Comment:
We usually write changelog entries in past tense -> Implemented

Reply:
I mean our release changelog entries :D somehow the changelog term became overloaded